src/org/zaproxy/zap/extension/pscanrulesBeta/resources/help/contents/pscanbeta.html

```diff
@@ -12,6 +12,22 @@
 
 <H2>Charset Mismatch</H2>
 This check identifies responses where the HTTP Content-Type header declares a charset different from the charset defined by the body of the HTML or XML. When there's a charset mismatch between the HTTP header and content body Web browsers can be forced into an undesirable content-sniffing mode to determine the content's correct character set.
+This check handles various conditions depending on the Threshold set for the scanner, as follows:
+<ul>
+  <li>All Thresholds:</li>
```

Comment:
I mean something like:
```HTML
<li>All Thresholds:
  <ul>
    <li>Header Versus Meta Content-Type Charset - The declaration in the HTTP Content-Type header doesn't match what is declared in a META Content-Type tag.</li>
    <li>Header Versus Meta Charset - The declaration in the HTTP Content-Type header doesn't match what is declared in a META Charset tag.</li>
  </ul>
</li>
```
So that the "ul" element is included in the "li" element.